src/api/erc20/Erc20ApiMock.ts

```diff
@@ -35,7 +35,6 @@ export class Erc20ApiMock implements Erc20Api {
   }
 
   public async allowance(tokenAddress: string, userAddress: string, spenderAddress: string): Promise<BN> {
-    await wait()
```

Comment:
Why remove the `wait()` here? I understand the wait already returns a noop when running the tests.

Reply:
Cause when running the app it does actually wait, and I didn't want to add a delay on the load of the app so it's faster